src/coreclr-debug/activate.ts

```diff
@@ -155,9 +157,16 @@ export class DebugAdapterExecutableFactory implements vscode.DebugAdapterDescrip
 
         // debugger has finished installation, kick off our debugger process
 
+        // Check for targetArchitecture
+        let targetArchitecture = "";
+        if (this.platformInfo.isMacOS())
+        {
+            targetArchitecture = _session?.configuration?.targetArchitecture ?? this.platformInfo.architecture;
```

Comment:
>this.platformInfo.architecture [](start = 80, length = 30)

Do we want to default to x64 until .NET 6 comes out? Or maybe do the work of running 'dotnet --info' and scaping the RID?